internal/cli/iam/projects/list.go

```diff
@@ -46,6 +47,7 @@ func (opts *ListOpts) initStore(ctx context.Context) func() error {
 }
 
 func (opts *ListOpts) Run() error {
+	time.Sleep(time.Minute)
```

Comment:
```suggestion
```

Reply:
This is just here temporarily (hence the *draft* PR) so that you can try it out for yourself without having to very quickly press CTRL-C. I will remove it before marking the PR ready for team review.